parts/kubernetesmasterresources.t

```diff
@@ -171,6 +171,63 @@
     },
     {
       "apiVersion": "[variables('apiVersionDefault')]",
+      "dependsOn": [
+       "[variables('vnetID')]"
```

Comment:
You will need something like:
{{if .MasterProfile.IsCustomVNET}}
         "[variables('nsgID')]",		         "[variables('nsgID')]",
 {{else}}		 {{else}}
          "[variables('vnetID')]",		          "[variables('vnetID')]",
  {{end}}		  {{end}}